Assets/LeapMotion/Core/Examples/Common Example Assets/Standard Fixed-White.shader

```diff
@@ -0,0 +1,46 @@
+﻿Shader "Leap Motion/Example Shaders/Standard Fixed-White" {
```

Comment:
This shader uses a different prefix ("Example Shaders" instead of "Examples") and doesn't seem to do anything the standard shader couldn't do?